zephyr/CMakeLists.txt

```diff
@@ -400,11 +401,36 @@ list(
       $<$<BOOL:${CONFIG_BACDL_ETHERNET}>:${BACNETSTACK_PORT}/ethernet.c>
       ${BACNETSTACK_PORT}/main.c
       ${BACNETSTACK_PORT}/mstimer-init.c
+      ${BACNETSTACK_PORT}/event.c
+      ${BACNETSTACK_PORT}/acc.c
       ${BACNETSTACK_PORT}/ai.c
+      ${BACNETSTACK_PORT}/ao.c
       ${BACNETSTACK_PORT}/av.c
-      ${BACNETSTACK_PORT}/event.c
```

Comment:
Clean up this list order to parallel the one in `$bacnet-stack/CMakeLists.txt`.  We need to make this one easy for non-Zephyr contributors to maintain.